deepsensor/data/loader.py

```diff
@@ -1226,7 +1302,7 @@ def sample_variable(var, sampling_strat, seed):
                 X_c_offrid_all = np.concatenate(X_c_offgrid, axis=1)
             Y_c_aux = (
                 self.sample_offgrid_aux(
-                    X_c_offrid_all, self.time_slice_variable(self.aux_at_contexts, date)
+                    X_c_offrid_all, self.time_slice_variable(self.aux_at_contexts, date), sample_patch_size
```

Comment:
We shouldn't need to spaitally slice offrid aux; this will happen implicitly because the context data used for sampling the `self.aux_at_contexts` xarray data will already have been spatially sliced.